deltacat/utils/placement.py

```diff
@@ -223,12 +228,13 @@ def _config(instance_cpus: int, instance_type: int, time_out: Optional[float] =
 				pg_res['CPU']+=node_detail['resources_total']['CPU']
 				pg_res['memory']+=v['memory']
 				pg_res['object_store_memory']+=v['object_store_memory']
-				pg_res['node_id'].append(node_id)
 		cluster_resources['CPU'] = int(pg_res['CPU'])
 		cluster_resources['memory'] = float(pg_res['memory'])
 		cluster_resources['object_store_memory'] = float(pg_res['object_store_memory'])
-		cluster_resources['node_id'] = pg_res['node_id']
-		pg_config=[opts,cluster_resources]
+		cluster_resources['node_id'] = node_res_keys # bundle_id
```

Comment:
If the cluster has 100 nodes but num_bundles is 90, we just want to round robin on 90 nodes. Hence, you may want to trim this list to take only num_bundles items or pass in only num_bundles items to this method.

Reply:
the current `node_res_keys` is only the available node ids in this placement group, so should be fine.